src/bundle/Controller/ContentOnTheFlyController.php

```diff
@@ -70,18 +86,24 @@ public function createContentAction(
         ContentType $contentType,
         Location $parentLocation
     ) {
+        if ($contentType->identifier == 'user') {
```

Comment:
It's not going to work. You have go rely on the fact that the content type has `ezuser` fieldtype or use AdminUI configuration which tells you what content types are reserved for users. You can ask @mikadamczyk or @ViniTou about the second approach.

Reply:
\EzSystems\EzPlatformAdminUi\Specification\ContentType\ContentTypeIsUser